docs/_documentations/remotedeploy-eclipse.md

```diff
@@ -45,52 +47,48 @@ Then complete the docker registryconnection details:
 3. Click the checkbox to desigate this registry as push registry
 4. Your dockerhub namespace (which is usually the same as your username)
 
-The connection will be tested to validate your credentials and are stored in a Kubernetes secret within the Codewind service.
+The connection is tested to validate your credentials, and these are then stored in a Kubernetes secret within the Codewind service.
 
 ## Create a new project
 
-Navigate to the Codewind panel and right click the deployment, then choose `create new project`:
+1\. Navigate to the Codewind panel, right-click the deployment, and then click `create new project`:
 
 ![Adding new remote project](./images/remoteeclipse/newProject.png)
 
-Select the template type, project name and your new project should be built and after a few seconds begin running in the cloud.
+2\. Select the template type, and name your project. Your new project is built and after a few seconds begins running in the cloud.
 
 ![Adding new remote project](./images/remoteeclipse/runningProject.png)
 
 ## Copying an existing local project to the cloud
 
-Copying an existing local project "myFirstNodeProject" over to the remote Codewind deployment.
-
-To begin :
+The folowing procedure copies an existing local project `myFirstNodeProject` over to the remote Codewind deployment:
 
-Right-click the remote deployment and click the "Add existing project button"
+1\. Right-click the remote deployment and click the `Add existing project button`:
 
 ![Add existing project](./images/remoteeclipse/addExistingProject.png)
 
-Navigate to the folder containing your local project files, or choose the previously created "myFirstNodeProject" from the workspace. Click next to proceed :
+2\. Navigate to the folder containing your local project files, or choose the previously created `myFirstNodeProject` from the workspace. Click `Next` to proceed:
 
 ![Add to Codewind](./images/remoteeclipse/existingProject.png)
 
-Codewind will ask you to confirm the project type identified as NodeJS:
+3\. Codewind prompts you to confirm the project type identified as NodeJS:
```

Comment:
To convey that it's not the name, "Node.js," we should probably put "NodeJS" in code phrase format.